packages/cli-platform-android/src/config/index.ts

```diff
@@ -132,10 +132,12 @@ export function dependencyConfig(
   const androidMkPath = userConfig.androidMkPath
     ? path.join(sourceDir, userConfig.androidMkPath)
     : path.join(sourceDir, 'build/generated/source/codegen/jni/Android.mk');
-  const cmakeListsPath = userConfig.cmakeListsPath
+  let cmakeListsPath = userConfig.cmakeListsPath
     ? path.join(sourceDir, userConfig.cmakeListsPath)
     : path.join(sourceDir, 'build/generated/source/codegen/jni/CMakeLists.txt');
-
+  if(process.platform == "win32") {
+    cmakeListsPath = cmakeListsPath.replaceAll("\\", "/")
```

Comment:
can you do the same for `androidMkPath` as well?

Reply:
Hi .. Sorry, i missed tracking this PR. The fix may not be required form ndk-build as it can handle the windows file paths i believe .. I will check it out and raise another PR if required.